omc3/model/accelerators/petra/job.p3.madx

```diff
@@ -0,0 +1,111 @@
+! --------------------------------------------------------------------------------------------------
```

Comment:
Maybe similar to LHC creation on the fly.. i.e. model creator? Less copy paste and more flexibility (tunes?)

Reply:
That's how those files were created.